### PR TITLE
Local Gatsby Source Plugin for retrieving files from repos

### PR DESCRIPTION
**Pull request by patricksimonian**
Several file changes are just formatting changes from Prettier js

### CODE REVIEW

File: app-web/plugins/gatsby-source-github-all/utils/constants.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const MEDIATYPES = {
+  md: 'text/markdown',
+  json: 'application/json',
+  yaml: 'text/yaml',
+  txt: 'text/html',
+  html: 'text/html',
+};
+
+const PROCESSABLE_EXTENSIONS = ['.md'];
```
Comment (ShellyXueHan) [inline app-web/plugins/gatsby-source-github-all/utils/constants.js]: do you need to export it?

File: app-web/plugins/gatsby-source-github-all/utils/github-api.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+ * b64 encoded string of the file contents into either raw string data or json
+ * @param {String} repo 
+ * @param {String} owner 
+ * @param {String} path 
+ * @param {String} token 
+ */
+const fetchFile = async (repo, owner, path, token) => {
+  try {
+    const result = await fetch(
+      `https://api.github.com/repos/${owner}/${repo}/contents/${path}`,
```
Comment (ShellyXueHan) [inline app-web/plugins/gatsby-source-github-all/utils/github-api.js]: use constant

Comment (patricksimonian) [reply app-web/plugins/gatsby-source-github-all/utils/github-api.js]: switched out for constant : )

File: app-web/plugins/gatsby-source-github-all/utils/constants.js (hunk 1 above)
Comment (patricksimonian) [reply app-web/plugins/gatsby-source-github-all/utils/constants.js]: yes i do. Good catch!

**Pull request merged**